src/main/java/com/locallampoon/fiveh/core/Game.java

```diff
@@ -78,84 +78,94 @@ private static Direction movementHelper(String newDirection) {
-                player.dropItem(droppedItem);
-                playerCurrentRoom.addItem(droppedItem);
-                break;
-            case "talk":
-                // TODO: Player needs a Talk method to talk with characters in rooms
+                case "get":
+                case "grab":
+                    String grabbedItem = UserInput.nounItemHelper(parsedCommandList, player);
+                    player.addItem(grabbedItem);
+                    playerCurrentRoom.removeItem(grabbedItem);
+                    break;
+                case "drop":
+                    String droppedItem = UserInput.nounItemHelper(parsedCommandList, player);
+                    player.dropItem(droppedItem);
+                    playerCurrentRoom.addItem(droppedItem);
+                    break;
+                default:
+                    System.out.println("Invalid action command");
+            }
+        } else if (parsedCommandList.size() == 1) {
+            switch (parsedCommandList.get(0)) {
+                case "talk":
+                    // TODO: Player needs a Talk method to talk with characters in rooms
 //                    player.talkWith(output.get(1));
 //                    break;
-            case "inspect":
-                // TODO: Player needs an Inspect method to items in room
+                case "inspect":
+                    // TODO: Player needs an Inspect method to items in room
 //                    player.inspectItem(output.get(1));
 //                    break;
-            case "fight":
-                Monster monster = playerCurrentRoom.getRoomMonster();
-                if (monster != null){
-                    player.attack(monster);
-                    if (monster.isDead()) {
-                        System.out.println(" You killed " + monster.getName());
+                case "fight":
+                    Monster monster = playerCurrentRoom.getRoomMonster();
+                    if (monster != null) {
+                        player.attack(monster);
+                        if (monster.isDead()) {
+                            System.out.println(" You killed " + monster.getName());
+                        } else {
+                            monster.attack(player);
+                        }
+                        if (player.isDead()) {
+                            System.exit(0);
+                        }
                     } else {
-                        monster.attack(player);
+                        System.out.println("There is no monster in this room");
                     }
-                    if (player.isDead()) {
-                        System.exit(0);
+                    break;
+                case "flee":
+                    player.flee(houseMap);
+                    break;
+
+                case "inventory":
+                case "i":
+                    player.printInventoryItems();
+                    break;
+                case "recruit":
```

Comment:
It seems that the `recruit` case belongs up in the if condition where `parsedCommandList.size() > 1`. Does this throw an error if you try to run `recruit` by itself, with no NPC specified?